rest/route/service.go

```diff
@@ -136,7 +136,6 @@ func AttachHandler(app *gimlet.APIApp, opts HandlerOpts) {
 	app.AddRoute("/patches/{patch_id}/raw").Version(2).Get().Wrap(checkUser, viewTasks).RouteHandler(makePatchRawHandler(sc))
 	app.AddRoute("/patches/{patch_id}/restart").Version(2).Post().Wrap(checkUser, submitPatches).RouteHandler(makeRestartPatch(sc))
 	app.AddRoute("/patches/{patch_id}/merge_patch").Version(2).Put().Wrap(checkUser, addProject, submitPatches, checkCommitQueueItemOwner).RouteHandler(makeMergePatch(sc))
-	app.AddRoute("/projects").Version(2).Get().Wrap(checkUser).RouteHandler(makeFetchProjectsRoute(sc))
```

Comment:
what's the reason for deleting this route?

Reply:
Oh right I meant to leave a comment: When going through the queries I found kind of a weird one for this route; I don't see any evidence that people are using this route, and I don't think we should be supporting it. For one thing, the authentication seems wrong. Yes we don't return project variables since we don't check if the user is a project admin, but these days really the whole project is supposed to consider viewProjectSettings. Also, this route just seems large and unnecessary. If one day there's a reason to just see all projects they have access to I think we can write a fresh route. (I'll also remove this from the documentation)

Reply:
I did a splunk search over the past week for path="/api/rest/v2/projects" and see a small number of requests to it, so we can't remove it without offering an alternative

Reply:
Oh sorry you're right, it looks like mana uses it. Weird that it's just been this week haha, I did a "previous month" search a week ago and there was nothing. I'll put this route back but do we know what they use it for (is it just for a list of project names)? I'd like to open a ticket to replace it.